modular_skyrat/modules/food_replicator/replicator_designs/replicator_medical.dm

```diff
@@ -0,0 +1,67 @@
+/datum/design/sutures
+	name = "sutures"
+	id = "slavic_suture"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/suture
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/mesh
+	name = "regenerative meshes"
+	id = "slavic_mesh"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/mesh
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/bruise_pack
+	name = "bruise packs"
+	id = "slavic_bruise"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 250)
+	build_path = /obj/item/stack/medical/bruise_pack
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/ointment
```

Comment:
and no, these shouldn't be attainable outside of the very rare rng chance for them to spawn

Reply:
:(
yeah sure ig ill remove those in the next commit